easybuild/tools/include.py

```diff
@@ -157,7 +157,7 @@ def verify_imports(pymods, pypkg, from_path):
 
 def is_software_specific_easyblock(module):
     """Determine whether Python module at specified location is a software-specific easyblock."""
-    return bool(re.search(r'^class EB_.*\(.*\):\s*$', read_file(module), re.M))
+    return bool(re.search(r'^class %s.*\(.*\):\s*$' % EASYBLOCK_CLASS_PREFIX, read_file(module), re.M))
```

Comment:
This regex might be too loosely specified. Since this is multiline it should be
```
r"^class %s([^(:]*)\(([^)]**)\):\s*$" % EASYBLOCK_CLASS_PREFIX
```
Or at least I think so...

Reply:
But perhaps keep that for a followup to not make this contain too many changes

Reply:
We touched that line anyway so might as well include it. Your regex seems to be fine, just removed the parens as we don't need any capture groups. I also added a comment to describe it